src/scenes/ObservationsRouter/details/ObservationDetailsRenderer.tsx

```diff
@@ -14,13 +14,18 @@ const ObservationDetailsRenderer =
   (props: RendererProps<TableRowType>): JSX.Element => {
     const { column, row, value } = props;
 
+    const has25mPlots = row.plantingSubzones
+      ?.flatMap((subzone: { monitoringPlots: any[] }) => subzone.monitoringPlots.flatMap((plot) => plot.sizeMeters))
+      .some((size: number) => size.toString() === '25');
+
```

Comment:
Perhaps this should be a static util function that receives subzones? It seems to be duplicated 3 times.

Reply:
It is used twice for subzones, so I added a util for that. The other checks the zone.